draft-ralston-mimi-linearized-matrix.md

```diff
@@ -220,6 +229,10 @@ A transport layer would describe a formal request/response structure for the mem
 requests should be able to be rejected by the owner server prior to the membership transition happening
 in the room, as the owner server may wish to apply additional checks for anti-abuse or similar.
 
+**XXX**: I still don't understand why we need membership APIs, rather than just sending the right state
+event to the server to invite/ban/kick/etc.  It just makes them look special-cased and makes the
+API look more bigger than it really is... --Matthew
```

Comment:
Membership has additional context above normal state handling. 

We're not saying here that one needs to have a transport layer difference between membership and state, just calling out that membership is slightly weird.

Reply:
We can rename the section if needed.